layouts/quote.vue

```diff
@@ -0,0 +1,40 @@
+<script setup>
+	import BaseLayout from './base-layout.vue';
+
+	const props = defineProps({
+    	author: String,
```

Comment:
Mind the indentation

Reply:
Dat is gek, in mijn editor is de indenting wel goed 🤔